adcc/AmplitudeVector.py

```diff
@@ -20,26 +20,39 @@
 ## along with adcc. If not, see <http://www.gnu.org/licenses/>.
 ##
 ## ---------------------------------------------------------------------
-import libadcc
 
 BLOCK_LABELS = ["s", "d", "t"]
 
 
+# TODO Extend AmplitudeVector to cases where only the doubles block is present
 class AmplitudeVector:
-    def __init__(self, *tensors):
+    def __init__(self, *tensors, ph=None, pphh=None):
         """Initialise an AmplitudeVector from some blocks"""
-        self.tensors = list(tensors)
+        # TODO ph and pphh are new-style constructors, they should
+        #      take over passing their tensors into a dict.
```

Comment:
See above, can we remove the `BLOCK_LABELS` and make `AmplitudeVector` a `dict` like you suggested?

Reply:
same thing ... that's one of the steps, I still have to do ;)